estate/views/estate_property_search_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="estate_property_search_view" model="ir.ui.view">
+        <field name="name">estate.property.search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Search...">
+                <field name="name" string="Title"/>
+                <field name="postcode"/>
+                <field name="expected_price"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="facades"/>
+                <!-- filter -->
+                <filter string="Available" name="available" domain="['|', ('state', '=', 'new'), ('state', '=', 'offer_received')]"/>
+                <!-- group by -->
+                <group expand="0" string="Group By">
+                    <filter string="Postcode" name="group_by_postcode" context="{'group_by':'postcode'}"/>
+                </group>
```

Comment:
I guess this is how it's shown in the tutorial (_because the other did it similarly hehe_). But the group is not necessary, the `context="group_by..` is sufficient to tell you're doing a group_by. If you try without, it should do the same
```suggestion
                <filter string="Postcode" name="group_by_postcode" context="{'group_by':'postcode'}"/>
```